src/journal/journalctl-misc.c

```diff
@@ -261,9 +261,13 @@ int action_list_namespaces(void) {
                 }
         }
 
-        r = table_print_with_pager(table, arg_json_format_flags, arg_pager_flags, !arg_quiet);
-        if (r < 0)
-                return table_log_print_error(r);
+        if (table_get_rows(table) <= 1 && !arg_quiet)
```

Comment:
table_isempty

Reply:
Hmm, if --quiet, `table_print_with_pager` shouldn't be called either?

Reply:
Or rather: output the legend after `table_print_with_pager`, so that if json output is requested we still output valid json